Calculator.py

```diff
@@ -0,0 +1,76 @@
+class math_funcs(object):
+    """This class include special math funcs"""
+
+    @staticmethod
+    def factorial(num: str):
+        """Doing factorial math function"""
+        fact = 1
+        for n in range(1, int(num) + 1):
+            fact = fact * n
+        return fact
+
+    @staticmethod
+    def average(num1: int, num2: int):
+        """"Doing average between two numbers"""
+        avg = (num1 + num2) / 2
+        return avg
+
+
+class Calculator(object):
+    """The class defines calculator with operations order"""
+
+    def __init__(self, list_of_exercise: list):
+        """
```

Comment:
Can you write for example the docstring for this class?

Reply:
go to your pycharm-> settings->search:"docstring"->python integrated tools->docstring format:"Epytext"

and from now on every time you will add  a comment it will give you a template with your params and etc... try it and tell me how is it